dora/src/baseline/cannon/codegen.rs

```diff
@@ -745,11 +830,13 @@ impl<'a, 'ast> CodeGen<'ast> for CannonCodeGen<'a, 'ast> {
                 | Bytecode::RetFloat(src)
                 | Bytecode::RetDouble(src) => self.emit_return_generic(&bytecode, *src),
                 Bytecode::RetVoid => {}
-                _ => panic!("bytecode {:?} not implemented", btcode),
+                _ => self.emit_epilog(),
```

Comment:
Why this? Why not crash for unknown bytecodes?

Reply:
This was a mistake.